config/initializers/ezid_credentials.rb

```diff
@@ -0,0 +1,16 @@
+# prod
+# EZID_DEFAULT_SHOULDER: ark:21985/N2
+# EZID_USER: user
+# EZID_PASSWORD: password
+# EZID_HOST: ezid.lib.purdue.edu\
+# EZID_PORT: 443
+# EZID_USE_SSL: true
+
+Ezid::Client.configure do |conf|
```

Comment:
We went back and forth on this for Arch, but we decided to get rid of the automatic fallback to the test credentials if the environment variables weren't present in production. https://github.com/nulib/institutional-repository/blob/master/config/initializers/doi_config.rb#L5

Since we're using the rails-config gem, we can have default ark/ezid values stored in the development (and staging?) environments and then use environment variables in production